### PR TITLE
new icon: raspberrypi (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by Kartstig**
- Adds [Raspberry Pi](https://www.raspberrypi.org/) icon
- From https://github.com/konpa/devicon/pull/102

### CODE REVIEW

**Comment by Thomas-Boi** [comment]
Hello there,
Recently, we just introduced an automation pipeline to Devicon which will help us create the icons. Before we can merge your changes in, there are a few things that we need to do first:

1. Merge the current devicon/develop branch into your PR branch. This contains the build script needed for us to build your icons.
2. Read the `CONTRIBUTING.md` for things to do before merging and review our icon standards.
3. Push your changes and/or branches. We will review them once this is done and merge your PR if everything goes well.


Thank you for contributing to this repo. We are sorry that we haven't accept any PR for awhile. Now that we have a build pipeline, we can merge PRs more quickly than before. 

We hope to see your icons in the Devicon repo soon :)

Cheers,
Devicon Team

**Comment by Thomas-Boi** [comment]
Hello there,

We notice that this PR has been inactive for awhile now. It would be a waste of your time and effort if we just close this PR and throw away the SVGs in here. Thus, 1 month from now, if this PR remains inactive/stale, we will fork your repo, fix any issues, and merge your icons ourselves.

Since GitHub tracks contributions by commits, you will still be credited. We also hope that you will help us merge this PR before the deadline comes.

Cheers,
Devicon Team

**Comment by Kartstig** [comment]
@amacado what would you like to do with this PR?

**Comment by amacado** [comment]
@Kartstig thanks for picking up! Feel free to update this pull request and fulfill our [contribution guidelines](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#overview-on-submitting-icons). Once done we can review, merge and ship it with the next release! :)

**Comment by Kartstig** [comment]
@amacado This should be good to go now (I think)

**Review by amacado — CHANGES_REQUESTED** [review]
We are missing a `font` version for this icon. At least one valid font version is required according to our guidelines.

File: icons/raspberrypi/raspberrypi-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="svg2"
+   xml:space="preserve"
+   width="3080.5601"
+   height="825.76001"
+   viewBox="0 0 3080.5601 825.76001"
```
Comment (amacado) [inline icons/raspberrypi/raspberrypi-original-wordmark.svg:15]: Please have a look at our guidelines regarding [svg requirements](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards). The when you update this pull request, those requirements should be checked by out bot.. Let's see :)

**Review by amacado — CHANGES_REQUESTED** [review]

**Comment by Thomas-Boi** [comment]
Hey @Kartstig,

Are you still planning to work on this? If not, I can take over and handle the rest.

**Comment by Kartstig** [comment]
@Thomas-Boi if you can pick this up, that would be great!

**Comment by Thomas-Boi** [comment]
> @Thomas-Boi if you can pick this up, that would be great!

No problem 👍. I'll handle this and the other PR then

**Comment by Thomas-Boi** [comment]
Hey @Kartstig, I'll be closing this PR. Don't worry, the icon is still being accepted at #609. Thank you for your work 😄

**Pull request closed without merging**